src/components/Annotation/AnnotationHeader.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import InlineDelete from '../../../third-party/components/InlineDelete';
+import UserLink from '../../../third-party/components/UserLink';
+import Timestamp from './Timestamp';
+import messages from './messages';
+
+import './AnnotationHeader.scss';
+
+type Props = {
+    id: string,
+    permissions: AnnotationPermissions,
+    createdAt?: string,
+    createdBy?: User,
+    onDelete?: Function
+};
+
+const AnnotationHeader = ({ id, permissions, onDelete, createdAt, createdBy }: Props) => {
+    const annotator =
+        !!createdBy && !!createdBy.name
+            ? createdBy
+            : {
```

Comment:
Moving this default to a constant could make this ternary a bit easier to read.

Reply:
Is the handling of a lack of user consistent with ui-elements and EUA? With the ticket I'm working on, based on the conversation I had with @JustinHoldstock there actually are Anonymous users of Box, so showing the Anonymous text when no user is passed in was misleading. Would that be true here as well?

Reply:
This case is specific to annotations because when a use creates their first annotation, we don't know who they are until the saved annotation response comes back from the API. Until that response comes back from the API, we display "Some User" as a placeholder and swap it out w/ the proper name later.